app/models/task.rb

```diff
@@ -2,6 +2,7 @@ class Task < ApplicationRecord
   ALLOWED_STATUSES = %w[not_started started finished draft].freeze
 
   belongs_to :project
+  belongs_to :user, optional: true
```

Comment:
It is okay, but you don't use it. You can leave it out, but I prefer to code only what I need in PR.

Reply:
u wrong 😢 🔫

Reply:
not u

Reply:
⚰️